src/coreclr/vm/mono/mono_coreclr.cpp

```diff
@@ -1297,38 +1297,12 @@ extern "C" EXPORT_API MonoType* EXPORT_CC mono_field_get_type_specific(MonoClass
 
 extern "C" EXPORT_API void EXPORT_CC mono_field_get_value(MonoObject *obj, MonoClassField *field, void *value)
```

Comment:
We could just remove these instead of asserting right?

Reply:
Exporting but asserting means Unity can load/retrieve these entry points. In some cases, we assert in Unity if an API is not available. So, removing is fine as long as we sync Unity side (which we should do longer term).